src/js/gi/components/AboutYourselfFields.jsx

```diff
@@ -0,0 +1,358 @@
+              <i className="fa fa-info-circle info-icons"></i>
+            </a>
+          </label>
+        </Dropdown>
+
+        <Dropdown
+            identifier="consecutive-service"
+            options={[
+              '0.8', '2+ years of consecutive service: 80%',
+              '0.6', '1+ year of consecutive service: 60%',
+              '0.4', '90+ days of consecutive service: 40%'
+            ]}
+            defaultOption="0.8"
+            alt="Select the length of your longest actice duty tour"
+            visible={this.state.dropdowns.consecutiveService}
+            onChange={this.handleDropdownChange}
+            showLabel={this.props.labels}>
+          <label htmlFor="consecutive-service">
+            Length of Longest Active Duty Tour:
+            <a onClick={() => {this.toggleModalDisplay('consecutiveService');}} className="info-icons">
+              <i className="fa fa-info-circle info-icons"></i>
+            </a>
+          </label>
+        </Dropdown>
+
+        <Dropdown
+            identifier="elig-for-post-gi-bill"
+            options={[
+              'yes', 'Eligilble for Post-9/11 Gi Bill',
+              'no', 'Not Eligilble for Post-9/11 Gi Bill'
+            ]}
+            defaultOption="no"
+            alt="Eligible for Post 9/11 GI Bill?"
+            visible={this.state.dropdowns.eligForPostGiBill}
+            onChange={this.handleDropdownChange}
+            showLabel={this.props.labels}>
+          {this.renderVREBenefitsMessage()}
+          <label htmlFor="elig-for-post-gi-bill">
+            Are you eligible for the Post-9/11 GI Bill?
+          </label>
+        </Dropdown>
+
+        <Dropdown
+            identifier="number-of-dependents"
+            options={[
+              '0', '0 Dependents',
+              '1', '1 Dependent',
+              '2', '2 Dependents',
+              '3', '3 Dependents',
+              '4', '4 Dependents',
+              '5', '5 Dependents'
+            ]}
+            defaultOption="0"
+            alt="Select the number of dependents you have"
+            visible={this.state.dropdowns.numberOfDependents}
+            onChange={this.handleDropdownChange}
+            showLabel={this.props.labels}>
+          <label htmlFor="number-of-dependents">
+            Number of Dependents
+          </label>
+        </Dropdown>
+
+        <Dropdown
+            identifier="online-classes"
+            options={[
+              'yes', 'All Online',
+              'no', 'None Online',
+              'both', 'Both online and on-campus'
+            ]}
+            defaultOption="no"
+            alt="Online? Yes/No"
+            visible={this.state.dropdowns.onlineClasses || !this.props.labels}
+            onChange={this.handleDropdownChange}
+            showLabel={this.props.labels}/>
+
+        <Modal onClose={() => {this.toggleModalDisplay('militaryStatus');}} visible={!!this.state.modals.militaryStatus}>
```

Comment:
Not sure if this is considered more idiomatic or not, but what I like to do is pass something like:

`this.toggleModalDisplay.bind(this, 'militaryStatus')`

Should end up being the same as creating the anonymous function.